listenbrainz_spark/recommendations/README.md

```diff
@@ -6,7 +6,9 @@ ListenBrainz collaborative filtering component to generate recommendations.
 
 The component uses collaborative filtering to recommend recordings to users based on their listening history. ListenBrainz uses Apache Spark to process the listening history of users and generate recommendations.
 
-**Note**:  The listening history and supporting data used by ListenBrainz is heavy for the local environment. We are trying to release small test datasets soon to make the development experience smooth.
+**Note**:  
+- The listening history and supporting data used by ListenBrainz is heavy for the local environment. We are trying to release small test datasets soon to make the development experience smooth.
```

Comment:
```suggestion
- The listening history and supporting data used by ListenBrainz is heavy for the local environment. Releasing small test datasets to make the development experience smoother is on our roadmap.
```

If we use words like soon and then don't do it for years or months, it gets a bit embarrassing so I try to avoid that